theforeman.org/pipelines/lib/foreman_infra.groovy

```diff
@@ -2,7 +2,15 @@ void git_clone_foreman_infra(args = [:]) {
     target_dir = args.target_dir ?: ''
 
     dir(target_dir) {
-        git url: 'https://github.com/theforeman/foreman-infra', poll: false
+        git url: 'https://github.com/theforeman/foreman_infra', poll: false
+    }
+}
+
+void git_clone_jenkins_jobs(args = [:]) {
```

Comment:
Added a new method for this repository given that some jobs need the puppet part of foreman-infra.